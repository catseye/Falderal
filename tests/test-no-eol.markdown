Falderal Test: Missing EOL at end of output
----------------------------

It makes no difference whether there is an EOL at the end of
the test output or not.

    -> Functionality "Echo" is implemented by
    -> shell command "python3 echo.py %(test-body-text)"

    -> Tests for functionality "Echo"

    | hello
    = hello

    | hi
    | hi
    = hi
    = hi

    -> Functionality "Echo, no newline" is implemented by
    -> shell command "python3 echo.py -n %(test-body-text)"

    -> Tests for functionality "Echo, no newline"

    | hello
    = hello

    | hi
    | hi
    = hi
    = hi
