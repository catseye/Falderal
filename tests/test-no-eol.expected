--------------------------------
Total test runs: 4, failures: 0
--------------------------------
